### DISCUSSION TITLE
Filter Setting for 3D Reconstructions

### DISCUSSION THREAD

**Discussion by MarkatLiu** [astra-toolbox/astra-toolbox#423] [Q&A]
Hi, I'm currently using the FDK shortscan method to recontruct the 3D helical scan data. According to the information, it is impossible to seting the filtering parameters for the FDK method. As a workaround, the filtering of the projection data and the 3D back projection can be performed seperately. Can you please help share examples on how to implement it in this way? Thanks.

**Comment by askorikov** [comment]
Hi! As far as I understand, the problem is not with the filtering step, but with the fact that FDK is not expected to work for helical data. See also #422.
